src/auth/certificate-upload.service.ts

```
@@ -0,0 +1,51 @@
+import { Injectable } from '@nestjs/common';
+import { ConfigService } from '@nestjs/config';
+import * as AWS from 'aws-sdk';
+import * as multer from 'multer';
```

ESLint

[error] 'multer' is defined but never used.

(@typescript-eslint/no-unused-vars)